apps/prairielearn/assets/stylesheets/pageLayout.css

```diff
@@ -113,7 +111,7 @@ body:has(div.app-container) {
   width: 4px;
   position: absolute;
   left: -0.75rem;
```

Comment:
I think at some point the horizontal spacing changed here, so this no longer shows on screen. For me, adjusting this left spacing to `-0.5rem` fixed things for me:

<img width="323" alt="Screenshot 2025-06-09 at 15 04 02" src="https://github.com/user-attachments/assets/ed369802-9a7c-4771-b634-938f98c629f0" />